risc-v-demo/sev-kit-reference-design/script_support/additional_configurations/smarthls/run_libero.bat

```diff
@@ -0,0 +1,21 @@
+@echo "off"
+SET prjDir=SEVPFSOC_H264
+SET HLS_PATH=script_support\additional_configurations\smarthls\hls_pipeline
+rmdir /S /Q %HLS_PATH%\hls_output 
+rmdir /S /Q %prjDir% 
+rmdir /S /Q component
+
+SET target=SMARTHLS:%HLS_PATH% SYNTHESIZE
+
+CALL libero.exe SCRIPT:MPFS_SEV_KIT_REFERENCE_DESIGN.tcl "SCRIPT_ARGS:%target%" logfile:partial_out.txt
+echo "------[ Identify ]--------"
+pushd "./SEVPFSOC_H264/synthesis/"
+synplify_pro.exe -licensetype synplifypro_actel -batch ..\..\script_support\additional_configurations\smarthls\synplify_enable_dbg_mode.tcl
```

Comment:
Why is Libero called with CALL but not synplify_pro?

Reply:
Resolved: https://github.com/MicrochipTech/fpga-hls-examples/pull/56/commits/efc36140eb5a18019730786e6ce7a9a4466c61b2